### PR TITLE
Webhook 2.0 docs

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…-tracking-plan-webhook**](https://docs-4ottdeorr.avo-dev.app/implementation/avo-tracking-plan-webhook) | [**…acking-plan/publishing**](https://docs-4ottdeorr.avo-dev.app/workspace/tracking-plan/publishing) |
|:-:|:-:|
|<a href="https://docs-4ottdeorr.avo-dev.app/implementation/avo-tracking-plan-webhook">Screenshot of /implementation/avo-tracking-plan-webhook</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ZYmTe3tTK7IG-YUDpCg-Jlc3j9JiDX61JCcthD-XLkAiRODbThJS1ffpiJY0GwELxYGWWiX6k1HYPNrk996XAg.png">(view full size)</a> |<a href="https://docs-4ottdeorr.avo-dev.app/workspace/tracking-plan/publishing">Screenshot of /workspace/tracking-plan/publishing</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/gyr5wpUguhbVeqlLetLNzBUOrc2Igaf4wsa_3WYuha4Ps2_mLn0f_o9iJs_oSRyiFxzfVzJl1QTVT0ajDeVmug.png">(view full size)</a> |







Commit b376bb3e5acdc7be02ab0f0684ad2e844c4b4dd7 (https://docs-4ottdeorr.avo-dev.app).

File: pages/workspace/tracking-plan/publishing.mdx [hunk 1]
```diff
@@ -217,3 +222,61 @@ The Tracking Plan will be delivered with a `POST` request containing the Trackin
...
+| events      | array(string) | Event ids belonging this category  |
+
+#### PublishInfo Model
+
+| Attribute       | Type             | Description                                                               |
+| :-------------- | :--------------- | :------------------------------------------------------------------------ |
+| branchId        | id               | Branch id                                                                 |
+| branchName      | string           | Branch name                                                               |
+| integrationId   | string           | ???                                                                       |
+| integrationName | string           | ???                                                                       |
```
Comment (alex-tpom6oh) [inline pages/workspace/tracking-plan/publishing.mdx:280]: @bjornj12 could you help with fields marked with `???`

File: pages/workspace/tracking-plan/publishing.mdx [hunk 2]
```diff
@@ -217,3 +224,61 @@ The Tracking Plan will be delivered with a `POST` request containing the Trackin
...
+| categories  | array(string)         | Category ids where this metric is included                 |
+| type        | string                | One of Proportion / EventSegmentation / Funnel / Retention |
+| items       | array of Nested Event | Event descriptions                                         |
+
+#### Nested Event Model
+
+| Attribute | Type       | Description |
+| :-------- | :--------- | :---------- |
+| eventId   | string     | Event id    |
+| where     | array(???) | ???         |
```
Comment (bjornj12) [inline pages/workspace/tracking-plan/publishing.mdx:244]:
```suggestion
| where     | array(whereFilter) |  Array of  where filters    |
```

File: pages/workspace/tracking-plan/publishing.mdx [hunk 3]
```diff
@@ -217,3 +224,61 @@ The Tracking Plan will be delivered with a `POST` request containing the Trackin
...
+| groupBy   | array(???) | ???         |
+
```
Comment (bjornj12) [inline pages/workspace/tracking-plan/publishing.mdx:246]:
```suggestion

#### Where filter

| Attribute     | Type              | Description |
|  :---------- | :------------ | :---------------------------------------------------  |
| propertyId  | string             |  property id                                                                       |
| operation    | string             | binary operation as string. ("equals", "notEquals" ...) | 
| values          | array(t)         | array of values of the same type                                    | 
```

File: pages/workspace/tracking-plan/publishing.mdx [hunk 4]
```diff
@@ -217,3 +224,61 @@ The Tracking Plan will be delivered with a `POST` request containing the Trackin
...
+| type        | string                | One of Proportion / EventSegmentation / Funnel / Retention |
+| items       | array of Nested Event | Event descriptions                                         |
+
+#### Nested Event Model
+
+| Attribute | Type       | Description |
+| :-------- | :--------- | :---------- |
+| eventId   | string     | Event id    |
+| where     | array(???) | ???         |
+| groupBy   | array(???) | ???         |
```
Comment (bjornj12) [inline pages/workspace/tracking-plan/publishing.mdx:245]:
```suggestion
| groupBy   | array(groupByFilter)| array of objects containing property ids to group by |
```
##### groupByFilter
| Attribute      | Type       | Description                                              |
| :----------- | :-------  | :-----------------------------------  |
| propertyId   | string     | property Id being used in group filter   |

File: pages/workspace/tracking-plan/publishing.mdx [hunk 5]
```diff
@@ -217,3 +224,61 @@ The Tracking Plan will be delivered with a `POST` request containing the Trackin
...
+| metrics     | array(string) | Metric ids belonging this category |
+| events      | array(string) | Event ids belonging this category  |
+
+#### PublishInfo Model
+
+| Attribute       | Type             | Description                                                               |
+| :-------------- | :--------------- | :------------------------------------------------------------------------ |
+| branchId        | id               | Branch id                                                                 |
+| branchName      | string           | Branch name                                                               |
+| integrationId   | string           | ???                                                                       |
```
Comment (bjornj12) [inline pages/workspace/tracking-plan/publishing.mdx:281]:
```suggestion
| integrationId   | string           |    Id of the publish integration (conistently the same for this integration) |
```

File: pages/workspace/tracking-plan/publishing.mdx (hunk 1 above)
Comment (bjornj12) [reply pages/workspace/tracking-plan/publishing.mdx:282]:
```suggestion
| integrationName | string           | Name of the publish integration (seen in Tracking Plan -> publishing -> <IntegrationName>                                                                      |
```

**Pull request merged**